### PR TITLE
Containerize RPM build to ensure CentOS-Stream9 RPMs

### LINKED ISSUE #535: Cannot use RPMs build locally on Fedora 37 on tmt tests
## Describe the bug

Building the RPMs locally on Fedora 37 creates a dependency on python 3.11 for bluechi-python. However, the tmt tests run on CentoOS Stream 9 containers which run python 3.9. As a result, the test container fails to build 

## To Reproduce

Follow the steps in the tests' README file for running locally built RPMs

## Expected behavior

The tests are able to run (regardless to success of failure of them)

### PR DESCRIPTION
**Pull request by ygalblum**
Add skipper environment configuration
Skip git config operation when running the containerized build
Add strato-skipper to the requirements.txt file
Update the tests README

Resolves: #535

### CODE REVIEW

File: Containerfile.bluechi-rpm-build [hunk 1]
```diff
@@ -0,0 +1,37 @@
+FROM quay.io/centos/centos:stream9
```
Comment (mkemel) [inline Containerfile.bluechi-rpm-build]: Did you try to work with https://quay.io/repository/bluechi/build-base ?
This way we would not need to maintain two `dnf install` statements.

Comment (ygalblum) [reply Containerfile.bluechi-rpm-build]: Great catch. I've changed it to start from this image. I still need to install `python-devel`. We could add it to the base image and remove this line. But, for now, this works

**Review by mkemel — APPROVED** [review]
LGTM

File: Containerfile.bluechi-rpm-build (hunk 1 above)
Comment (mkemel) [reply Containerfile.bluechi-rpm-build]: Awesome, thanks! Merging

**Pull request merged**